.rustfmt.toml

```diff
@@ -10,15 +10,46 @@ reorder_imports = true
 reorder_modules = true
 tab_spaces = 4
 use_field_init_shorthand = true
+use_small_heuristics = "Default"
 use_try_shorthand = true
-edition = "2018"
 
-# unstable options (for one-off fixes and CI)
-merge_imports = true
-wrap_comments = true
+# unstable options
+binop_separator = "Front"
+blank_lines_lower_bound = 0
+blank_lines_upper_bound = 1
+brace_style = "PreferSameLine"
+combine_control_expr = false
 comment_width = 100
 condense_wildcard_suffixes = true
+control_brace_style = "AlwaysSameLine"
+empty_item_single_line = true
+enum_discrim_align_threshold = 50
+fn_args_density = "Tall"
+fn_single_line = true
+force_multiline_blocks = true
+format_doc_comments = true
+format_macro_bodies = true
+format_macro_matchers = true
```

Comment:
`false` is the default value
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#format_macro_matchers

I figured if we're formatting the bodies, why not the matchers too?